conf/services/tax-credits-renewals.cy.yml

```diff
@@ -0,0 +1,11 @@
+serviceName: Tax Credits Renewals
+serviceHeaderName: Gwasanaeth Adnewyddu Credydau Treth
+serviceDescription: |
+  Gall unrhyw un sy’n cael credydau treth ar hyn o bryd ddefnyddio’r gwasanaeth Adnewyddu Credydau Treth i adnewyddu eu
+  hawliad yn ystod y cyfnod adnewyddu rhwng mis Ebrill a mis Awst.
+serviceDomain: www.tax.service.gov.uk
+serviceUrl: /tax-credits-service/renewals
+contactFrontendServiceId: tax-credits-renewals
+complianceStatus: noncompliant
+statementCreatedDate: 2021-03-22
+statementLastUpdatedDate: 2021-03-22
```

Comment:
1. This YAML is missing the mandatory property: `statementVisibility`
2. The property `serviceHeaderName` is no longer used and should be removed